routes/projects/delete.js

```diff
@@ -1,12 +1,12 @@
 /*
- * DELETE /project/:username/:projectName
+ * DELETE /projects/:username/:projectName
  * Deletes the given project for the given user
```

Comment:
I assume this cascades to also delete all files associated with the project, as well as published resources?  If so, call it out here.